apis/dialapi/dialapi.yaml

```diff
@@ -1100,14 +1349,14 @@ paths:
             to contact us. In the meantime, try refreshing.
           content: {}
       deprecated: false
-      security: 
+      security:
         - bearer: []
   /dialcode/v1/list:
     post:
       tags:
       - DIAL APIs
       summary: List DIAL code
-      description: 
+      description:
```

Comment:
Please check the formatting of the description and the bullet points. They are appearing in a single paragraph instead of bullet points.

Reply:
Dont know what to do for formatting. I have added space to see if it resolves the problem. Please verify